servers/physics_3d/shape_3d_sw.cpp

```diff
@@ -101,10 +101,7 @@ const Map<ShapeOwner3DSW *, int> &Shape3DSW::get_owners() const {
 	return owners;
 }
 
-Shape3DSW::Shape3DSW() {
-	custom_bias = 0;
-	configured = false;
-}
+Shape3DSW::Shape3DSW() {}
```

Comment:
Empty constructors could be removed.

Reply:
I was only able to move empty constructors to .h files, because probably due virtual destructors, build fails.